src/java.desktop/share/classes/javax/swing/AbstractButton.java

```diff
@@ -2146,9 +2146,10 @@ protected void init(String text, Icon icon) {
 
 
     /**
-     * This is overridden to return false if the current <code>Icon</code>'s
-     * <code>Image</code> is not equal to the
-     * passed in <code>Image</code> <code>img</code>.
+     * If the button icon for present button state is either <code>null</code>
+     * or not an <code>ImageIcon</code> with an <code>Image</code> equal to the
```

Comment:
assuming the intent is correct, "for the current button state" reads better than "for present button state"

Reply:
"for present button state" text replaced with "for the current button state".